src/server.js

```diff
@@ -17,6 +18,13 @@ dotenv.config();
 const app = express();
 app.use(express.json());
 
+const allowedOrigin = process.env.FRONT_API_URL.replace(/\/$/, "");
+app.use(
+  cors({
+    origin: allowedOrigin,
+  })
+);
+
```

Comment:
```suggestion
app.use(
  cors({
    origin: process.env.FRONT_API_URL.replace(/\/$/, "")
  })
);
```

`allowedOrigin`이 재사용되는 것이 아니라 따로 상수화를 진행하지 않아도 될 것 같은데 이 부분에 대해 어떻게 생각하시는지 궁금합니다. @Eun0713 @faber-h

Reply:
기능적으로는 동일하지만, 추후 Whisper 서버 등 다양한 도메인이 추가될 가능성을 고려하여 allowedOrigin을 상수로 분리했습니다.
이렇게 하면 여러 출처를 배열로 관리하도록 쉽게 확장할 수 있어, 초기 단계부터 유지보수성을 확보될 수 있을 것 같습니다 :)

Reply:
`process.env.FRONT_API_URL?.replace(/\/$/, "")`처럼 문자열을 조정하는 처리가 포함되어 있어서 인라인보다는 상수로 분리하는 게 의도 전달이나 재사용 측면에서 더 명확하다고 생각어요~

물론 재윤님 말씀처럼 인라인으로 써도 되지만, 이 코드 상에서는 상수화가 더 읽기 쉬운 코드가 될 거라고 판단했습니다.

Reply:
희주님과 다은님께서 말씀해주신 바와 같이 향후 Whisper 서버같은 다른 도메인이 추가될 경우 배열 형태로 allowedOrigin에서 관리하면 되겠네요. 두 분 모두 좋은 말씀 감사드립니다!